src/node.cc

```diff
@@ -2497,36 +2501,95 @@ struct node_module* get_linked_module(const char* name) {
   return mp;
 }
 
-// DLOpen is process.dlopen(module, filename).
+struct DLib {
+  void* handle_;
+  std::string filename_;
+  std::string errmsg_;
+  int flags_;
+
+  void* Handle(struct DLib *lib) {
+    return lib->handle_;
+  }
+
+  const char* GetError(struct DLib *lib) {
+    return lib->errmsg_.c_str();
+  }
+
+#ifdef __POSIX__
+
+  #define DLOPEN_DEFAULT_FLAGS RTLD_LAZY;
```

Comment:
Don't indent preprocessor directives (not portable) and drop the semicolon.

Namespace it by calling it `NODE_DLOPEN_DEFAULT_FLAGS` or better, turn it into `static const int kDefaultFlags = RTLD_LAZY;`.

Reply:
Oh, of course.